src/jdk.jlink/share/classes/jdk/tools/jmod/JmodOutputStream.java

```diff
@@ -54,15 +55,17 @@
      * This method creates (or overrides, if exists) the JMOD file,
      * returning the the output stream to write to the JMOD file.
      */
-    static JmodOutputStream newOutputStream(Path file) throws IOException {
+    static JmodOutputStream newOutputStream(Path file, LocalDateTime date) throws IOException {
         OutputStream out = Files.newOutputStream(file);
         BufferedOutputStream bos = new BufferedOutputStream(out);
-        return new JmodOutputStream(bos);
+        return new JmodOutputStream(bos, date);
     }
 
     private final ZipOutputStream zos;
-    private JmodOutputStream(OutputStream out) {
+    private final LocalDateTime   date;
```

Comment:
```suggestion
    private final LocalDateTime date;
```

No need for extra spaces.